CPAC/utils/utils.py

```diff
@@ -1218,8 +1218,20 @@ def check_config_resources(c):
     else:
         num_ants_cores = 1
 
+    # Now check OMP
+    if c.num_omp_threads is None:
+        num_omp_cores = 1
+    elif c.num_omp_threads > c.maxCoresPerParticipant:
+        err_msg = 'Number of threads for OMP: %d is greater than the ' \
+                    'number of threads per subject: %d. Change this and ' \
+                    'try again.' % (c.num_omp_threads,
+                                    c.maxCoresPerParticipant)
+        raise Exception(err_msg)
+    else:
+        num_omp_cores = c.num_omp_threads
+
     # Return memory and cores
-    return sub_mem_gb, num_cores_per_sub, num_ants_cores
+    return sub_mem_gb, num_cores_per_sub, num_ants_cores, num_omp_cores
```

Comment:
This will cause `ValueError: too many values to unpack (expected 3)` without handling everywhere `check_config_resources` is called (I think just these 2 places in longitudinal):
https://github.com/FCP-INDI/C-PAC/blob/f4791c181284d5bd8b702a8502c35a4f1efc427e/CPAC/longitudinal_pipeline/longitudinal_workflow.py#L111-L112 https://github.com/FCP-INDI/C-PAC/blob/f4791c181284d5bd8b702a8502c35a4f1efc427e/CPAC/longitudinal_pipeline/longitudinal_workflow.py#L1399-L1400

Reply:
Good catch! Resolved!